tracing-attributes/src/attr.rs

```diff
@@ -356,4 +361,5 @@ mod kw {
     syn::custom_keyword!(target);
     syn::custom_keyword!(name);
     syn::custom_keyword!(err);
+    syn::custom_keyword!(ret);
```

Comment:
personally, my preference would be to use `return` rather than `ret`, for consistency with the `return` keyword --- the abbreviation feels like it would just make things more confusing for the user. I'm open to being convinced otherwise, though, if most folks think that saving three letters here is really important...

also, the field name is `return` rather than `ret`, so it's a little surprising that they are different.

Reply:
I'm reluctant to use `return` for the sake of syntax highlighting. `return` in this image is highlighted as a keyword, 
but ideally `err` and `return` should be in the same color as they are both arguments. ![2021-11-17-180810_7680x2160_scrot](https://user-images.githubusercontent.com/57856193/142170750-bf3646d5-d47b-40bd-a275-5511f5301458.png)

Reply:
hmm, that's a good point; I hadn't considered that...